src/libraries/System.Private.CoreLib/src/System/Threading/LazyThreadSafetyMode.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Threading
     public enum LazyThreadSafetyMode
     {
         /// <summary>
-        /// This mode makes no guarantees around the thread-safety of the <see cref="Lazy{T}"/> instance.  If used from multiple threads, the behavior of the <see cref="Lazy{T}"/> is undefined.
```

Comment:
Thanks for your interest in cleaning things up. However, two spaces after a period is a fairly common convention that was used historically, resulting in over 9000 places it's done in .cs files in this repo. I don't believe it's worth the churn to try to get them to all conform to a single-space-after-a-period rule, and I don't think it's worthwhile fixing comments one file at a time.